go/stellar/stellar.go

```diff
@@ -250,14 +250,19 @@ func LookupRecipient(ctx context.Context, g *libkb.GlobalContext, to RecipientIn
 	return res, err
 }
 
+type DisplayAmount struct {
+	Amount   string
+	Currency string
+}
```

Comment:
It's a little confusing to have 'amount' only sometimes include the unit. How about `DisplayBalance`?